Microsoft.Toolkit.Uwp.UI.Controls/Graph/PeoplePicker/PeoplePicker.xaml

```diff
@@ -0,0 +1,95 @@
+                                          Margin="0,0,30,0"
+                                          HorizontalAlignment="Right"
+                                          AutomationProperties.Name="Progress"
+                                          FlowDirection="LeftToRight"
+                                          IsActive="False" />
+                        </Grid>
+                        <Grid>
+                            <StackPanel>
+                                <ListBox Name="SelectionsListBox"
+                                         AutomationProperties.Name="Selections"
+                                         ItemsSource="{TemplateBinding Selections}">
+                                    <ListBox.ItemTemplate>
+                                        <DataTemplate>
+                                            <Grid AutomationProperties.Name="{Binding DisplayName}">
+                                                <Grid.ColumnDefinitions>
+                                                    <ColumnDefinition Width="Auto" />
+                                                    <ColumnDefinition Width="*" />
+                                                    <ColumnDefinition Width="Auto" />
+                                                </Grid.ColumnDefinitions>
+                                                <controls:ProfileCard Grid.Row="0"
+                                                                      Grid.Column="0"
+                                                                      DisplayMode="LargeProfilePhotoLeft"
+                                                                      UserId="{Binding Id}" />
+                                                <TextBlock Grid.Row="0"
+                                                           Grid.Column="1" />
+                                                <Button Grid.Row="0"
+                                                        Grid.Column="2"
+                                                        Width="48"
+                                                        Height="48"
+                                                        Margin="0"
+                                                        HorizontalAlignment="Right"
+                                                        AutomationProperties.Name="Remove from selection"
+                                                        Background="Transparent"
+                                                        BorderThickness="0"
+                                                        Command="{Binding DataContext.DeleteItemCommand, ElementName=PeoplePickerContainer}"
+                                                        CommandParameter="{Binding Id}"
+                                                        FlowDirection="LeftToRight"
+                                                        Tag="{Binding UserPrincipalName}">
```

Comment:
Tag doesn't seem to be used elsewhere?  What's this for?

Reply:
@michael-hawker , Thanks, we've cleaned up the unused code.